libauk/src/main/java/com/bitmark/libauk/util/SecurityFile.kt

```diff
@@ -0,0 +1,47 @@
+package com.bitmark.libauk.util
+
+import android.content.Context
+import androidx.fragment.app.FragmentActivity
+import com.bitmark.libauk.storage.SecureFileStorageImpl
+import io.reactivex.Single
+import java.io.File
+import java.util.UUID
+
+class SecurityFileUtil {
+    fun readAllFiles(context: Context, nameFilterFunc: (String) -> Boolean): Single<Map<String, ByteArray>> {
```

Comment:
the function seems unused and I think the logic is not correct. Should we remove or update the correct logic for it?